spec/abci++/abci++_methods_002_draft.md

```diff
@@ -468,25 +469,25 @@ When a validator _p_ enters Tendermint consensus round _r_, height _h_, in which
 
 * **Request**:
 
-    | Name   | Type  | Description                                                                   | Field Number |
-    |--------|-------|-------------------------------------------------------------------------------|--------------|
-    | hash   | bytes | The header hash of the proposed block that the vote extension is to refer to. | 1            |
-    | height | int64 | Height of the proposed block (for sanity check).                              | 2            |
+    | Name   | Type  | Description                                                                    | Field Number |
+    |--------|-------|--------------------------------------------------------------------------------|--------------|
+    | hash   | bytes | The header hash of the proposed block that the vote extensions is to refer to. | 1            |
+    | height | int64 | Height of the proposed block (for sanity check).                               | 2            |
 
 * **Response**:
 
-    | Name              | Type  | Description                                   | Field Number |
-    |-------------------|-------|-----------------------------------------------|--------------|
-    | vote_extension    | bytes | Optional information signed by by Tendermint. | 1            |
+    | Name            | Type  | Description                                   | Field Number |
+-----------------|-------------------|-------|-----------------------------------------------|--------------|
```

Comment:
I think this should actually be (it doesn't render the table properly currently):
```suggestion
|-------------------|-------|-----------------------------------------------|--------------|
```